services/postgres/healthcheck.sh

```diff
@@ -10,4 +10,23 @@ fi
 
 PG_READY_CMD="pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DATABASE}"
 
-( ${PG_READY_CMD} && sleep 10 && ${PG_READY_CMD} ) || exit 1
+# check that the database is accessible at all, with a 2-second
```

Comment:
Just curious what you were planning to use this for. An UptimeRobot check? Periodic CI ping to notify us of downtime? Did someone request it?

Reply:
It was because the database becomes "ready" intermittently during boot; if it signals that it's ready too soon, the API container would start to run migration scripts against it and fail.

Now that I think about it, though, this probably isn't necessary now that the boot script uses a sentinel file to indicate it's *actually* ready. I'll review it and remove it if it works without it. Thanks for the heads up!

Reply:
FYI, I ended up removing it in favor of just waiting on the sentinel file and it seems to work fine.